.github/workflows/ci.yml

```diff
@@ -30,4 +30,5 @@ jobs:
       - run: ./setup.sh
       - run: rustup component add rustfmt --toolchain nightly
       - run: rustup component add clippy --toolchain nightly
+      - run: rustup component add audit --toolchain nightly
```

Comment:
Do we still use nightly?

Reply:
We can even remove it from the setup script.

Reply:
Removed